app/src/main/kotlin/com/wire/android/ui/calling/incoming/IncomingCallScreen.kt

```diff
@@ -112,8 +114,8 @@ private fun IncomingCallContent(
         sheetPeekHeight = dimensions().defaultIncomingCallSheetPeekHeight,
         sheetContent = {
             CallOptionsControls(
-                isMuted = callState.isMuted,
-                isCameraOn = callState.isCameraOn,
+                isMuted = callState.isMuted ?: true,
```

Comment:
Don't we also need to check if it is group call or OneOnOne call?

As for when joining a:
`Group Call` -> microphone is `muted`
`OneOnOne Call` -> microphone is `un-muted`

Reply:
And the same logic (but reversed) for camera?

Reply:
This is just a default value given, at launch time we are updating it based  exact type (in observeCallState)